src/Input/InputLabel.js

```diff
@@ -15,10 +15,10 @@ export const styleSheet = createStyleSheet('MuiInputLabel', theme => ({
     position: 'absolute',
     left: 0,
     top: 0,
-    transform: 'translate(0, 18px) scale(1)',
+    transform: `translate(0, ${theme.spacing.unit * 5}px) scale(1)`,
   },
   shrink: {
-    transform: 'translate(0, 0px) scale(0.75)',
+    transform: `translate(0, ${theme.spacing.unit * 2 + 2.5}px) scale(0.75)`,
```

Comment:
Without this hack, other inputs labels flicker when focusing one. Couldn't find better solution.
Only if we switch from transitions to animation.